Python/Product/PythonTools/PythonTools/Environments/VirtualEnv.cs

```diff
@@ -425,5 +425,13 @@ IInterpreterRegistryService service
             }
             return libPath;
         }
+        
+        internal static bool IsVirtualEnvironment(string prefixPath) {
+            if(string.IsNullOrEmpty(prefixPath)) {
+                return false;
+            }
+
+            return (File.Exists(Path.Combine(prefixPath, "pyvenv.cfg")) || File.Exists(Path.Combine(FindLibPath(prefixPath) ?? "", "orig-prefix.txt")));
```

Comment:
If `FindLibPath` is `null`, you'll be checking for "orig-prefix.txt" in the current directory (which is unpredictable). Best to do this in multiple statements, don't do the `File.Exists` if folder is `null`.